CleverTap/Runtime/Native/UnityNativeWrapper/UnityNativeEventQueueManager.cs

```diff
@@ -9,12 +9,12 @@ internal class UnityNativeEventQueueManager {
         private readonly UnityNativeBaseEventQueue _userEventsQueue;
         private readonly UnityNativeBaseEventQueue _raisedEventsQueue;
 
-        internal UnityNativeEventQueueManager(UnityNativeCoreState coreState, UnityNativeNetworkEngine networkEngine, UnityNativeDatabaseStore databaseStore) {
+        internal UnityNativeEventQueueManager(UnityNativeCoreState coreState, UnityNativeNetworkEngine networkEngine, UnityNativeDatabaseStore databaseStore, UnityNativeEventValidator eventValidator) {
```

Comment:
Why do we need an eventValidator here? It is not needed by the queues so it is not really used.

Reply:
Removed